LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2024
+Copyright (c) 2024 Mahdi, Mayito & Uzair
```

Comment:
# Frontend review

- [ ] Your frontend readme does not feature any tests, therefore, kindly remove these instructions:

![image](https://github.com/mahdinoori2000/healthcare-app-backend/assets/57421823/b3b3ef48-60bd-484c-a93a-7122739b8f9f)


![image](https://github.com/mahdinoori2000/healthcare-app-backend/assets/57421823/c03cd07e-c53b-417e-88f4-516352bdb03b)